website/static/css/pages/wiki-page.css

```diff
@@ -281,7 +281,7 @@
 }
 
 .wmd-button > span {
-    background-image: url(../../../addons/wiki/static/pagedown-ace/wmd-buttons.png);
+    background-image: url(../../../static/public/vendor/pagedown/wmd-buttons.png);
```

Comment:
Since you're using the publicly-served URL, you don't need to use a relative URL. You can just do: 

```css
background-image: url(/static/public/vendor/pagedown/wmd-buttons.png);
```

I will make this change when I merge.